docs/release_notes/v0.16.0.md

```diff
@@ -0,0 +1,35 @@
+# New in 0.16.0 (2022-04-08)
+
+In this release, we have refactored our destVI model:
+
+1. Bug fix in cell type amortization, which leads to on par performance of cell type amortization with free parameter for cell type.
```

Comment:
I would add in parenthesis the parameter name and the argument so people know what arguments this corresponds to